core/src/test/scala/com/rackspace/com/papi/components/checker/wadl/WADLCheckerSpec.scala

```diff
@@ -5411,6 +5505,59 @@ class WADLCheckerSpec extends BaseCheckerSpec with LogAssertions {
+            XPath("/tst:a/@id"), XPath("/tst:a/@stepType"), XSL, XSD, SetHeaderAlways("Warning"), SetHeaderAlways("Warning"), Accept)
+    assert (checker, Start, URL("a"), URL("b"), Method("POST"), ReqType("(application/xml)(;.*)?"), ContentFail)
+    assert (checker, Start, URL("a"), URL("b"), Method("POST"), ReqType("(application/xml)(;.*)?"), WellXML, ContentFail)
+    assert (checker, Start, URL("a"), URL("b"), Method("POST"), ReqType("(application/xml)(;.*)?"), WellXML, XPath("/tst:a","Expecting the root element to be: tst:a"), ContentFail)
+    assert (checker, Start, URL("a"), URL("b"), Method("POST"), ReqType("(application/xml)(;.*)?"), WellXML, XPath("/tst:a","Expecting the root element to be: tst:a"),
+            XPath("/tst:a/@id"), ContentFail)
+    assert (checker, Start, URL("a"), URL("b"), Method("POST"), ReqType("(application/xml)(;.*)?"), WellXML, XPath("/tst:a","Expecting the root element to be: tst:a"),
+            XPath("/tst:a/@id"), XPath("/tst:a/@stepType"), XSL, ContentFail)
+  }
+
+  scenario("The WADL contains a POST  operation accepting valid xml with elements specified and multiple required plain params, and a single XSL transform (disabled warn headers)") {
+    Given ("a WADL that contains a POST operation with elemets specified and multiple plain params and a single XSL transform (disabled warn headers)")
+    val inWADL =
+      <application xmlns="http://wadl.dev.java.net/2009/02"
+                   xmlns:tst="http://www.rackspace.com/repose/wadl/checker/step/test"
+                   xmlns:rax="http://docs.rackspace.com/api">
+        <grammars>
+            <include href="src/test/resources/xsd/test-urlxsd.xsd"/>
+        </grammars>
+        <resources base="https://test.api.openstack.com">
+           <resource path="/a/b">
+               <method name="POST">
+                  <request>
+                      <representation mediaType="application/xml" element="tst:a">
+                         <param name="id" style="plain" path="/tst:a/@id" required="true"/>
+                         <param name="stepType" style="plain" path="/tst:a/@stepType" required="true"/>
+                         <rax:preprocess href="src/test/resources/xsl/testXSL1.xsl"/>
+                      </representation>
+                  </request>
+               </method>
+           </resource>
+        </resources>
+    </application>
+    register("test://app/src/test/resources/xsd/test-urlxsd.xsd",
+             XML.loadFile("src/test/resources/xsd/test-urlxsd.xsd"))
+    register("test://app/src/test/resources/xsl/testXSL1.xsl",
+             XML.loadFile("src/test/resources/xsl/testXSL1.xsl"))
+    val config = TestConfig(false, false, true, true, true, 1, true, true, true)
+    config.enableWarnHeaders = false
+    val checker = builder.build (inWADL, config)
+    assert(checker, "in-scope-prefixes(/chk:checker/chk:step[@type='XPATH' and @match='/tst:a']) = 'tst'")
```

Comment:
and a `Then` above this?

- `Then("the following assertions should also hold:")`